### PR TITLE
Add slider and some styling

### PR DESCRIPTION
**Pull request by Razzeee**

### CODE REVIEW

File: public/main.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
+.input-range-container {
```
Comment (Razzeee) [inline public/main.css:1]: We need to cleanup this file somewhere down the line

File: src/Colors.elm [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module Colors exposing (..)
```
Comment (Razzeee) [inline src/Colors.elm:1]: This is mostly unused, but I think this would be a good idea

File: src/Components.elm [hunk 3]
```diff
@@ -167,33 +169,54 @@ player :
...
-        , column [ height fill, width (fillPortion 2) ]
-            [ row [ height (px 70), width fill, Background.color (rgb 0.2 0.2 0.2), alignBottom, padding 10, spacing 30 ]
-                [ image [ alignLeft ]
-                    { src = "https://via.placeholder.com/70"
-                    , description = "Hero Image"
-                    }
-                , el [ Font.color (Element.rgb 0.6 0.6 0.6), Font.size 18, Font.family [ Font.typeface "Open Sans", Font.sansSerif ] ] (text currentlyPlaying.title)
-                , el [ alignRight, Font.color (Element.rgb 0.6 0.6 0.6), Font.size 18, Font.family [ Font.typeface "Open Sans", Font.sansSerif ] ] (text (String.fromInt currentlyPlaying.duration))
-                ]
+        , currentlyPlayingColumn currentlyPlaying volumeSlider
```
Comment (Razzeee) [inline src/Components.elm:184]: I'm passing `volumeSlider` for now. This needs to be changed.

**Comment by Razzeee** [comment]
One thing that's weird is, that the slider seems to have an invisible box, where the slider button is. It's actually bigger then the bar itself.

File: public/main.css (hunk 1 above)
Comment (ionif) [reply public/main.css:1]: probably should do it sooner than later, would rather keep it nice than deal with css spaghetti 😄

File: src/Components.elm (hunk 3 above)
Comment (ionif) [reply src/Components.elm:184]: what are you thinking of changing it to?

Comment (ionif) [reply src/Components.elm:184]: ah I see nvm

**Pull request merged**